lib/rubocop/cop/sequel/irreversible_migration.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Sequel
+      # IrreversibleMigration looks for methods inside a `change` block that cannot be reversed.
+      class IrreversibleMigration < Base
+        # https://sequel.jeremyevans.net/rdoc/files/doc/migration_rdoc.html#label-A+Basic+Migration
+        VALID_CHANGE_METHODS = %i[
+          create_table
+          create_join_table
+          create_view
+          add_column
+          add_index
+          rename_column
+          rename_table
+          alter_table
+          add_column
+          add_constraint
+          add_foreign_key
+          add_primary_key
+          add_index
+          add_full_text_index
+          add_spatial_index
+          rename_column
+          set_column_allow_null
+        ].freeze
+
+        MSG = 'Avoid using this method inside a `change` block. Use a `down` block instead.'
```

Comment:
I think you can use `%<stuff>s` in the message to substitute the offending method name using `format` method

see our rubocop_rules/disallow_methods.rb for example